includes/discovery/sensors/current/ict-psu.inc.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * ict-psu.inc.php
+ *
+ * LibreNMS current sensor discovery module for ICT Digital Series Power Supply
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 Lorenzo Zafra
+ * @author     Lorenzo Zafra<zafra@ualberta.ca>
+ */
+
+// Output Current
+// SNMPv2-SMI::enterprises.39145.11.8.0 = STRING: "0.4" -- outputCurrent
+
+$outputCurrent_oid = '.1.3.6.1.4.1.39145.11.8.0';
```

Comment:
You've used the numeric OID all over but have provided a mib, it's much cleaner to use the name from the MIB if you can.